src/script.js

```diff
@@ -0,0 +1,39 @@
+const UI_MODE_SAVE_KEY = 'ui_mode'
+const UI_DARK = 'dark'
+const UI_LIGHT = 'light'
+
+// toggle
+const htmlToggle = document.querySelector('.toggle')
+htmlToggle.onclick = () => onClickToggle()
+
+// initial state
+const readStorage = localStorage.getItem(UI_MODE_SAVE_KEY)
+const initialState = readStorage ?? UI_LIGHT
+initialize(initialState)
+
+function initialize(state) {
+   console.log('Made with ❤️ in Sublime Text')
+   const toggle = state === UI_DARK ? true : false
```

Comment:
` const toggle = state === UI_DARK`

(github didn't allow me to make a suggestion for some reason)

Reply:
ooo that is tidy 💪 i updated!